Sources/TuistLoader/Utils/ManifestFilesLocator.swift

```diff
@@ -51,6 +55,11 @@ public final class ManifestFilesLocator: ManifestFilesLocating {
         return traverseAndLocate(at: at, appending: subPath)
     }
 
+    public func locateDependencies(at: AbsolutePath) -> AbsolutePath? {
+        let subPath = RelativePath("\(Constants.tuistDirectoryName)/\(Manifest.dependencies.fileName(at))")
```

Comment:
This confuses me a bit. Does `Manifest.dependencies.fileName(at)` return the name of the manifest file? I think you need to read the value from [this attribute](https://github.com/tuist/tuist/blob/master/Sources/TuistLoader/Models/Manifest.swift#L28)

Reply:
But `fileName(at)` is a selector, isn't it? Should we add a test for this function to make sure we are getting the expected value?

Reply:
I added some tests for this method in `ManifestFilesLocatorTests` in this PR. All pass.

Reply:
you can check them [here](https://github.com/tuist/tuist/blob/a61725fa8143fc8df643aa9b4f387d60e431e319/Tests/TuistLoaderTests/Utils/ManifestFileLocatorTests.swift#L158)

Reply:
you can check them [here](https://github.com/tuist/tuist/blob/a61725fa8143fc8df643aa9b4f387d60e431e319/Tests/TuistLoaderTests/Utils/ManifestFileLocatorTests.swift#L158)